tests/framework/builder.py

```diff
@@ -210,10 +216,28 @@ def build_from_snapshot(self,
                                          guest_ip=iface.guest_ip,
                                          netmask_len=iface.netmask,
                                          tapname=iface.tap_name)
-        response = vm.snapshot.load(mem_file_path=jailed_mem,
-                                    snapshot_path=jailed_vmstate,
-                                    diff=diff_snapshots,
-                                    resume=resume)
+        if utils.compare_versions(vm.firecracker_version, '1.0.0') >= 0:
```

Comment:
Here we need to do `utils.compare_versions(vm.firecracker_version, '1.0.0') > 0`, right?
Now one possible solution would be to tag Firecracker to point to the next release which has not proven to be a very accurate way (see v.0.26.0).
One other option would be to modify the `compare_versions` to be able to check dirty versions (i.e the ones that are not actual releases).
Then we could use `test_microvm.version.get_from_api()` to get the full firecracker version (including the dirty info).
Something like:
```
    is_first_dirty = False; is_second_dirty = False

    # code to see if any of the versions contain dirty info  (i.e if anything follows X.Y.Z)
    # we should not allow for both versions to be dirty; not sure if the dirty info reveals order info
    .....
    if is_first_dirty and is_second_dirty:
        raise ValueError
    if is_first_dirty:
        return 1
    if is_second_dirty:
        return -1
    return 0
```

Reply:
Cool, we can go like this

Reply:
I ended up adding this in this PR after all, because when using binaries built from the local firecracker version, `git describe` is not aware of the version bump (it looks at the latest tag).